src/components/Media/Media.tsx

```diff
@@ -29,6 +30,7 @@ export const Media = (props: MediaAllProps) => {
         parallax = false,
         metrika,
         fullScreen,
+        fullscreen,
```

Comment:
lets comment all this fullScreen and fullscreen cases to find and fix them later

or create utility function which would manage with both flags